src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/AbstractDSLStrategy.java

```
@@ -35,6 +37,30 @@ public AbstractDSLStrategy(int tabs, IDescriptor descriptor) {
 	public AbstractDSLStrategy(int tabs, IDescriptor descriptor, boolean shouldInitChildren) {
 		this.tabs = tabs;
 		this.propertyDescriptor = descriptor;
+		propertiesToBeSkipped.add("actions");
+		propertiesToBeSkipped.add("configVersion");
+		propertiesToBeSkipped.add("submoduleCfg");
+		propertiesToBeSkipped.add("doGenerateSubmoduleConfigurations");
+		propertiesToBeSkipped.add("canRoam");
+		propertiesToBeSkipped.add("caseSensitive");
+		propertiesToBeSkipped.add("followSymlinks");
```

Why do we skip this property? Is it from `archiveArtifacts`? This property could be set using Dynamic DSL. Silently ignoring properties that could be translated into DSL doesn't sound like a good idea 🤔

---

Yeah this whole feature was added before I started working on this project. I can remove the entire feature if it doesn't make sense. 

I suspect the intention was to not give annoying error flags on properties that were not supported.

---

I think what i'll do is
* Move this list into a txt file
* Audit the list and only add properties that are truly not supported in any way (ie can't be finagled with configure blocks, like for promoted builds)
* Instead of skipping altogether, return them with a message saying these properties are not supported by DSL (versus not being supported by the translator)

Which will require a bit more out of scope work so I think I'll just remove this feature from this PR and add it in at a later date.